README.md

```diff
@@ -165,7 +165,7 @@ Hurray! You successfully made a contribution!
 
 - ### **J**
 
-  - [Jatin Kumar](https://github.com/Jatinkumar30)
+  - [Jatin Kumar]( https://github.com/Jatinkumar30 )
```

Comment:
@Jatinkumar30 Don't Add a Space between the GitHub Profile Link.

Reply:
Sure , I've removed the spacing now.